tests/ui/parser/use-colon-as-mod-sep.stderr

```diff
@@ -11,18 +11,24 @@ error: expected `::`, found `:`
    |
 LL | use std:fs::File;
    |        ^ help: use double colon
+   |
```

Comment:
If the repeated note here is bad, I can either back this out or add `note_once` to the derive API